src/utils/httpRequestParser.ts

```diff
@@ -119,7 +119,7 @@ export class HttpRequestParser implements IRequestParser {
 
             let graphQlPayload = {
                 query: body,
-                variables: JSON.parse(variables.toString())
+                variables: variables ? JSON.parse(variables.toString()) : {}

```

Comment:
Or the variables should be omitted?

Reply:
variables can't be omitted. Some servers have schema validations, that check for both `{query,variables}`